recipe/meta.yaml

```
@@ -12,51 +15,81 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 2
+  skip: true            # [win32]
+  skip: true            # [win and py<35]
+  skip: true            # [cuda_compiler_version not in (undefined, "None", "9.2")]
+  number: {{ number }}
 
-requirements:
-  build:
-    - cmake
-    - ninja
-    - make  # [unix]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-  host:
-    # directly pin boost-cpp as we also seem to directly include boost symbols
-    # in the Python modules.
-    - arrow-cpp {{ version }}
-    - boost-cpp
-    - cython
-    - numpy 1.16.*
-    - python
-    - setuptools
-    - setuptools_scm
-    - six
+outputs:
+  - name: pyarrow
+    script: build-pyarrow.sh  # [not win]
+    script: bld.bat           # [win]
+    version: {{ version }}
+    build:
+      number: {{ number }}
+      string: "py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ number }}_{{ build_ext }}"
+      skip: true              # [win32]
+      skip: true              # [win and py<35]
+      skip: true              # [cuda_compiler_version not in (undefined, "None", "9.2")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+      activate_in_script: True
 
-  run:
-    - {{ pin_compatible('numpy', lower_bound='1.16') }}
-    - boost-cpp
-    - futures  # [py==27]
-    - glog
-    - pandas
-    # empty parquet-cpp metapackage, force old versions to be uninstalled
-    - parquet-cpp 1.5.1.*
-    - python
-    - six
-    - snappy
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - make                # [unix]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # pyarrow does not require nvcc but it needs to link against libraries in arrow-cpp=*=*cuda
+        - {{ compiler("cuda") }}  # [cuda_compiler_version is not undefined and cuda_compiler_version == '9.2']
+      host:
+        # directly pin boost-cpp as we also seem to directly include boost symbols
+        # in the Python modules.
+        - arrow-cpp {{ version }} *_{{ build_ext }}
```

Would just add `arrow-cpp-proc` here too.

```suggestion
        - arrow-cpp {{ version }}
        - arrow-cpp-proc * {{ build_ext }}
```

---

Fixed.